### PR TITLE
perfschema: simplify performance schema

### PR DESCRIPTION
**Pull request by jackysp**
## What have you changed? (mandatory)
Performance schema only has some compatible function in TiDB now. This pr
1. Rename VirtualTable to perfSchemaTable, and move it from package table to perfschema.
1. Simplify the implement of perfSchema. It simply returns empty results.
After this pr we can remove MemoryTable due to some perf schema tables also depend on it.

## What is the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

Unit tests.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No.

## Does this PR affect tidb-ansible update? (mandatory)
No.

## Does this PR need to be added to the release notes? (mandatory)
No.

PTAL @coocood @tiancaiamao

### CODE REVIEW

File: perfschema/perfschema.go [hunk 1]
```diff
@@ -18,18 +18,18 @@ import (
 	"github.com/pingcap/tidb/table"
 )
 
-// PerfSchema exports for test.
-type PerfSchema struct {
+// perfSchema exports for test.
```
Comment (tiancaiamao) [inline perfschema/perfschema.go]: Update comment

File: perfschema/perfschema_test.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 PingCAP, Inc.
```
Comment (tiancaiamao) [inline perfschema/perfschema_test.go]: s/2017/2018

File: perfschema/tables_test.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package perfschema_test
```
Comment (tiancaiamao) [inline perfschema/tables_test.go]: Two different package name can exist in a single directory? 
I see there is a `perfschema` in perfschema_test.go

Comment (jackysp) [reply perfschema/tables_test.go]: It is OK for test files only.

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**